### DISCUSSION TITLE
Size

### DISCUSSION THREAD

**Discussion by mdouble** [Blazor-Diagrams/Blazor.Diagrams#390] [Q&A] [answered]
First of all, thank you for the good work. It really is an amazing library. However, I seem to have a very simple problem.
I'm having trouble displaying elements in different sizes. Is there a way to display some nodes larger than others? I've already changed the "Size" property, but unfortunately it didn't have the desired effect. I am grateful for any help.

**Comment by arpanparpan** [comment]
Hi. Do it with css on your div. 
The customnodesample describes this good. 
If you want different sizes i would recommend registering two different models with their respective widget.

**Reply by mdouble** [reply]
Thank you very much for your answer. I am gonna try this. However, have you got an idea what the size property is for. Actually I want to change the size dynamically and it looked like a convenient way to do this.

**Reply by zHaytam** [reply, marked as answer]
Hello, currently the Size node is getting filled after the node renders and we get it with ResizeObserver. So it's more of a readonly property that is handled by the library.

In the future, I'm planning to have another library on top of this one with known shapes where you can control the size and everything.

Just create a custom node model with another size property and set that to what you want, then in your razor component use it.
